src/System.Linq/tests/SelectTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public class SelectTests
+    {
+        [Fact]
+        public void Select_SourceIsNull_ArgumentNullExceptionThrown()
+        {
+            IEnumerable<int> source = null;
+            Func<int, int> selector = i => i + 1;
+
+            AssertException<ArgumentNullException>(() => source.Select(i => i + 1));
```

Comment:
I didn't realize it had that dependency (or that this would manifest).  Probably because I'm dealing with the Parallel version, so the package was imported for other tests.

I think I prefer using the xunit-provided version, even with that caveat.  You should only need to add it as a project package - you don't have to list it as a `using`.  Although I don't know if I count as "on the team" for this purpose...